src/components/AddBook.js

```diff
@@ -1,16 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
 
-const AddBook = () => (
-  <div className="addbook-content">
-    <h4>ADD NEW BOOK</h4>
-    <div className="add-book-btns">
-      <form>
-        <input type="text" />
-        <input type="text" />
-        <input type="submit" value="ADD BOOK" />
+const AddBook = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
 
-      </form>
+  const submitBookToStore = (e) => {
+    e.preventDefault();
+
+    const newBook = {
+      id: 4, // make sure it's unique
```

Comment:
- [ ] It will be great to generate a unique ID for every book that is created. In your implementation, you have hard-coded `4` as the ID. What this means is that for every book that is created, you will have 4 as the ID. Remember for  this project to be approved, the ID should be unique